testing/framework/TestSCons.py

```diff
@@ -1079,7 +1087,16 @@ def java_get_class_files(self, dir):
                     result.append(os.path.join(dirpath, fname))
         return sorted(result)
 
-    def Qt_dummy_installation(self, dir: str='qt') -> None:
+    def ninja_binary(self):
```

Comment:
Curious if this a method for efficiency reasons, or if it would be better as just a variable at a top level like is done with the path to Python?

Reply:
Good point.
I'll change to that and update the branch.